e2e/playwright/text-to-cad-tests.spec.ts

```diff
@@ -689,3 +691,749 @@ test(
+      await expect(
+        page.getByText('Successfully deleted "my-project-name"')
+      ).toBeVisible()
+      // Project DOM card has this test id
+      await expect(page.getByTestId(projectName)).not.toBeVisible()
+    }
+  )
+
+  test(
+    'Home Page -> Text To CAD -> New Project -> Stay in home page -> Accept -> should navigate to file',
+    { tag: '@electron' },
+    async ({ context, page }, testInfo) => {
+      const u = await getUtils(page)
+      const projectName = 'my-project-name'
+      const prompt = '2x2x2 cube'
+      await mockPageTextToCAD(page)
+
+      // open commands
+      await page.getByTestId('command-bar-open-button').click()
+
+      // search Text To CAD
+      await page.keyboard.type('Text To CAD')
+      await page.keyboard.press('Enter')
+
+      // new project
+      await page.keyboard.type('New project')
+      await page.keyboard.press('Enter')
+
+      // write name
+      await page.keyboard.type(projectName)
+      await page.keyboard.press('Enter')
+
+      // prompt
+      await page.keyboard.type(prompt)
+      await page.keyboard.press('Enter')
+
+      await page.getByRole('button', { name: 'Accept' }).click()
+
+      await expect(page.getByTestId('app-header-project-name')).toBeVisible()
+      await expect(page.getByTestId('app-header-project-name')).toContainText(
+        projectName
+      )
+      await expect(page.getByTestId('app-header-file-name')).toBeVisible()
+      await expect(page.getByTestId('app-header-file-name')).toContainText(
+        '2x2x2-cube.kcl'
+      )
+
+      await u.openFilePanel()
+      await expect(
+        page.getByTestId('file-tree-item').getByText('2x2x2-cube.kcl')
+      ).toBeVisible()
+    }
+  )
+
+  test(
+    'Home Page -> Text To CAD -> Existing Project -> Stay in home page -> Reject -> should delete single file',
+    { tag: '@electron' },
+    async ({ context, page }, testInfo) => {
+      const projectName = 'my-project-name'
+      const prompt = '2x2x2 cube'
+      await mockPageTextToCAD(page)
+
+      // Create and navigate to the project then come home
+      await createProject({ name: projectName, page, returnHome: true })
+
+      await expect(page.getByText('Your Projects')).toBeVisible()
+
+      await expect(page.getByText('1 file')).toBeVisible()
+
+      // open commands
+      await page.getByTestId('command-bar-open-button').click()
+
+      // search Text To CAD
+      await page.keyboard.type('Text To CAD')
+      await page.keyboard.press('Enter')
+
+      // new project
+      await page.keyboard.type('Existing project')
+      await page.keyboard.press('Enter')
+
+      // write name
+      await page.keyboard.type(projectName)
+      await page.keyboard.press('Enter')
+
+      // prompt
+      await page.keyboard.type(prompt)
+      await page.keyboard.press('Enter')
+
+      await expect(page.getByRole('button', { name: 'Reject' })).toBeVisible()
+      await expect(page.getByText('2 file')).toBeVisible()
```

Comment:
In future, this should instead make use of the `homePage` fixture, like this example here https://github.com/KittyCAD/modeling-app/blob/main/e2e/playwright/feature-tree-pane.spec.ts#L73-L81